action.yml

```diff
@@ -30,11 +34,12 @@ runs:
         path: ${{ runner.temp }}/sonar-scanner-cli-${{ inputs.scannerVersion }}-${{ runner.os }}-${{ runner.arch }}
         key: sonar-scanner-cli-${{ inputs.scannerVersion }}-${{ runner.os }}-${{ runner.arch }}
     - name: Install Sonar Scanner CLI
-      if: steps.sonar-scanner-cli.outputs.cache-hit != 'true'
+      if: ${{ env.NO_CACHE == 'true' || steps.sonar-scanner-cli.outputs.cache-hit != 'true' }}
```

Comment:
I needed to add this `NO_CACHE` parameter to ignore previously cached scanners.
Clearing the cache via [gh cache delete](https://cli.github.com/manual/gh_cache_delete) is possible but requires repo-scope permissions (see here).

Reply:
LGTM, but out of curiosity, is `NO_CACHE` env variable something standard?

Reply:
No, it's a made-up name, inspired by the "no_cache" in Docker :-)
If it looks too "generic", we can change it for something more context-specific, also to avoid potential collisions with users using `NO_CACHE` in their workflow for something else.
Would you prefer, for example, `NO_SCANNER_CACHE` or `IGNORE_SCANNER_CACHE`?

Reply:
I am leaving as it is for now. 
By staying with a generic name `NO_CACHE`, we can reuse it later on to ignore any other cache, including, for example, the build wrapper (which is going to be useful when we are going to forward C++ actions to this action). 
Moreover, the probability of collision is fairly limited I'd say, and we should be able to change it at any point in time with no consequence (we don't expose it to the users in README, examples nor docs).